pkg/router/f5/f5.go

```diff
@@ -977,7 +977,12 @@ func (f5 *f5LTM) AddVtep(ipStr string) error {
 		Name:     macAddr,
 		Endpoint: ipStr,
 	}
-	return f5.post(url, payload, nil)
+	err = f5.post(url, payload, nil)
+	if err != nil && err.(F5Error).httpStatusCode != HTTP_CONFLICT_CODE {
```

Comment:
We are constructing this error type.
https://github.com/rajatchopra/origin/blob/f5_fdb/pkg/router/f5/f5.go#L407
The field httpStatusCode is always filled up either with nil value '0' if the request failed without obtaining the http status code or with the actual status code here:
https://github.com/rajatchopra/origin/blob/f5_fdb/pkg/router/f5/f5.go#L430